### ISSUE TITLE
Make process/application/executable consistent accross the software

### ISSUE THREAD

**Issue by Ileca** [evilsocket/opensnitch#621]
You have:

* the **Applications** tab
* **Application rules** in the rule tab's left bar, which is bad naming because your rules might have nothing to do with processes and be only about host, PID, port, etc. No application here. I suggest something along the line of "Time scope" but I am not very good in English so your call... >.>
* **Process** in the event tab
* **Applications** when editing a rule
* **From this executable** when editing a rule
* **from this executable** in the pop-up

See, you have three different names for the same concept. It confused me a lot when I started using OpenSnitch because I wondered if they had different meanings, which is not the case. An "executable"... Isn't that a Windows thing?
I know OpenSnitch wants to be an application firewall but I suggest to rename all those instances as *process*. The "processes" tab, "from this process", etc. At least, to be consistent across the whole software. Thanks.

**Comment by gustavo-iniguez-goya** [comment]
> Application rules in the rule tab's left bar, which is bad naming because your rules might have nothing to do with processes and be only about host, PID, port, etc. No application here. I suggest something along the line of "Time scope" but I am not very good in English so your call... >.>

You're right. Maybe "Application rules" -> "Rules" ? And then we could create another item below "Rules" called "Applications" or something similar.

> An "executable"... Isn't that a Windows thing?

I'm not sure 100% but I think that it was called "process" before "executable", but someone suggested to change it to "executable". If someone else is reading this issue maybe they can provide their point of view.

**Comment by Ileca** [comment]
"Rules" is redundant with the title of the tab. If you want something general, you could use "Filter" but Nodes below is also a filter/rule. That's why the title must refer to the nature of permanent/temporary. That's why I proposed "time scope". 
Then, I remembered you do have a column about that. It's called "duration". If you want to be consistent, this is what you should use imo.

As for executable, we use binary mostly. We use the term executable in the context of permissions or as an adjective. I don't remember seeing it used as a noun like here. Anyway, whatever you choose, I think executable shouldn't be on the list. Application fails when talking about what we talked recently, applications using other processes such as Python to make connections. Still better than executable.
